packages/components/src/rating/style.scss

```diff
@@ -5,6 +5,7 @@
 	vertical-align: middle;
```

Comment:
One other small thing we could take care of while we're here to vertically align the ratings in tables:

```suggestion
	display: flex;
	align-items: middle;
	justify-content: flex-end;
```

<img width="880" alt="Screen Shot 2019-06-19 at 9 19 37 AM" src="https://user-images.githubusercontent.com/10561050/59730053-87dfdb80-9273-11e9-92ca-7abec8ad48fe.png">


Not sure if this should only be applied to ratings within tables.

Reply:
This breaks the review panel

<img width="404" alt="Screen Shot 2019-06-19 at 2 07 00 PM" src="https://user-images.githubusercontent.com/1922453/59731769-9b9f3800-929b-11e9-8f88-c96a25f297fc.png">

It seems vertically aligned as is in Chrome. Are you seeing something different?

<img width="756" alt="Screen Shot 2019-06-19 at 2 07 58 PM" src="https://user-images.githubusercontent.com/1922453/59731813-c38e9b80-929b-11e9-9d3f-6f845dd90c8d.png">

Reply:
I think in your attached screenshot it's slightly off-center as well (just by a few pixels) causing the row height to shift.

We could apply the flexbox fix to only ratings in tables or skip entirely if you think it's not a big deal.

```scss
.woocommerce-table__item .woocommerce-rating {
	display: flex;
	align-items: middle;
	justify-content: flex-end;
}
```

Reply:
Still getting oddness on this. Since this example code isn't getting adopted, I think we should skip adding new css rules just in case.

![Screen Shot 2019-06-20 at 2 16 19 PM](https://user-images.githubusercontent.com/1922453/59813587-2c424a80-9366-11e9-9283-0602a8081ac0.png)